app/src/androidTest/java/com/github/wnder/NewPictureTesting.java

```diff
@@ -42,6 +45,22 @@ public static void createPic() throws InterruptedException {
         }
     }
 
+    // Clean db
+    @AfterClass
+    public static void deleteTestPic() {
+        try {
+            sleep(2000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
What does this stand for ? There's probably a good reason but just to be sure.

Reply:
As most other test, there is issues with the asynchrony of a few functions so the picture might be deleted before some test finish.
These issues should slowly resolve with the usage of the new PicturesDatabase.